clippy_lints/src/comparison_chain.rs

```diff
@@ -64,6 +64,10 @@ impl<'tcx> LateLintPass<'tcx> for ComparisonChain {
             return;
         }
 
+        if parent_node_is_if_const_fn(cx, expr) {
```

Comment:
It would be better to use `in_constant` in utills: https://github.com/rust-lang/rust-clippy/blob/1c3a3e7dc62ae3b4b84cc00c067789dae1d722c9/clippy_utils/src/lib.rs#L199